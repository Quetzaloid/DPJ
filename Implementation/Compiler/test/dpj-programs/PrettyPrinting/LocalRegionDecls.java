/**
 * Local region declarations
 * 
 * @author Rob Bocchino
 */
public class LocalRegionDecls<region P> {
    public void m() {
	region r1;
	LocalRegionDecls<Local:r1> x;
    }
}
