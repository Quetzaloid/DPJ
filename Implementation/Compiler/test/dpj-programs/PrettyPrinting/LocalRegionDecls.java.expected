public class LocalRegionDecls<region P> {
    
    public LocalRegionDecls() {
        super();
    }
    
    public void m() {
        region r1;
        LocalRegionDecls<Local:r1> x;
    }
}